src/main/flight/alt_hold.c

```diff
@@ -16,124 +16,51 @@
  */
 
 #include "platform.h"
+#include "math.h"
 
 #ifdef USE_ALT_HOLD_MODE
 
-#include "math.h"
 #include "build/debug.h"
-
+#include "common/maths.h"
 #include "config/config.h"
-#include "fc/runtime_config.h"
 #include "fc/rc.h"
+#include "fc/runtime_config.h"
+#include "flight/autopilot.h"
 #include "flight/failsafe.h"
-#include "flight/imu.h"
-#include "flight/pid.h"
 #include "flight/position.h"
-#include "flight/position_control.h"
-#include "sensors/acceleration.h"
 #include "rx/rx.h"
 
 #include "alt_hold.h"
 
-static float pidIntegral = 0.0f;
 static const float taskIntervalSeconds = HZ_TO_INTERVAL(ALTHOLD_TASK_RATE_HZ); // i.e. 0.01 s
 
 altHoldState_t altHoldState;
 
-float altitudePidCalculate(void)
+void altitudePidCalculate(void)
```

Comment:
```suggestion
void calcAltitudePid(void)
```

Reply:
Good idea.
However, I'll name it `controlAltitude`, since now what it really does is pass parameters to the `altitudeControl` function `autopilot.c`, where the PIDs are calculated and the motors get driven.
Adjusted some comments related to this.

Reply:
In the refactoring for position hold I tidy up a lot of these comments also.